tslearn/shapelets.py

```diff
@@ -560,9 +560,10 @@ def locate(self, X):
                [0],
                [0]])
         """
-        X = check_dims(X, self._X_fit_dims)
+        check_is_fitted(self, '_X_fit_dims')
```

Comment:
We should probably do this refactor on all places where `check_is_fitted` is used.

Reply:
OK, I'll check.